pkg/platform/abstract/platform.go

```diff
@@ -1160,6 +1163,47 @@ func (ap *Platform) validateProjectHasNoRelatedResources(projectMeta *platform.P
 	return nil
 }
 
+func (ap *Platform) enrichCodeEntryType(functionConfig *functionconfig.Config) {
```

Comment:
🥇on effort
but add on another PR

this PR -> add feature
another PR -> deprecate feature